towerlib/entities/schedule.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# File: schedule.py
+#
+# Copyright 2020 Ilija Matoski
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+#  of this software and associated documentation files (the "Software"), to
+#  deal in the Software without restriction, including without limitation the
+#  rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+#  sell copies of the Software, and to permit persons to whom the Software is
+#  furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+#  all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#  DEALINGS IN THE SOFTWARE.
+#
+
+"""
+Main code for Schedule.
+
+.. _Google Python Style Guide:
+   http://google.github.io/styleguide/pyguide.html
+
+"""
+
+from towerlib.towerlibexceptions import (InvalidJobType,
+                                         InvalidVerbosity)
+from .core import Entity, JOB_TYPES, VERBOSITY_LEVELS
+from .inventory import Inventory
+
+
+class Schedule(Entity):
+    """Models the schedule entity of ansible tower."""
+
+    def __init__(self, tower_instance, data):
+        Entity.__init__(self, tower_instance, data)
+
+    @property
+    def rrule(self):
```

Comment:
wouldn't it be friendlier to actually call the variable "recurrence_rule"?

Reply:
@SayantanKhanra10 already fixed this. But yes I do agree it's friendlier.